packages/frontend/electron/src/main/onboarding.ts

```diff
@@ -62,6 +69,20 @@ async function createOnboardingWindow(additionalArguments: string[]) {
     }, 300);
   });
 
+  // When moved to another screen, resize to fit the screen
+  browserWindow.on('moved', () => {
+    const position = browserWindow.getPosition();
+    const size = browserWindow.getSize();
+    const currentDisplay = screen.getDisplayNearestPoint({
+      x: position[0] + size[0] / 2,
+      y: position[1] + size[1] / 2,
+    });
```

Comment:
do we need to use the same code for initial size/pos as well?

Reply:
Yes I think, it's uncertain which screen the window will open on.